sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeBatchActionsOperation.cs

```diff
@@ -262,7 +262,7 @@ private async ValueTask<Response> UpdateStatusAsync(bool async, CancellationToke
                     _actionSucceeded = update.Value.Tasks.Completed;
                     _totalActions = update.Value.Tasks.Total;
 
-                    if (update.Value.Status == TextAnalyticsOperationStatus.Succeeded || update.Value.Status == TextAnalyticsOperationStatus.PartiallySucceeded)
+                    if (update.Value.Status == TextAnalyticsOperationStatus.Succeeded || update.Value.Status == TextAnalyticsOperationStatus.PartiallySucceeded || update.Value.Status == TextAnalyticsOperationStatus.PartiallyCompleted)
```

Comment:
oh interesting, for curiosity, what is the difference between `PartiallySucceeded ` and `PartiallyCompleted`?

Reply:
this is a duplicate, service has unified it to PartiallyCompleted.